src/Backend/Modules/Extensions/Layout/Templates/DetailModule.html.twig

```diff
@@ -95,7 +95,7 @@
       </div>
     </div>
   {% endif %}
-  {% if showExtensionsInstallModule %}
```

Comment:
I think the `showExtensionsInstallModule` check should still remain there too. We only want to show the button if both the module isn't installed and the user has rights to install modules.